src/Search.vue

```diff
@@ -4,7 +4,7 @@ import { useData } from "vitepress";
 // @ts-ignore
 import Index from "./module/index.js";
 
-const { localePath } = useData();
+const { localeIndex } = useData();
```

Comment:
Could you try to import both and check witch one has data?
Like const path = localePath || localeIndex
This way it won't brake CI from those already using it.

An 
//Todo: delete deprecate code on final version comment would be nice then.

Reply:
Also please increase the version of the package 🙂

Reply:
i also meet this problem